File: src/main/java/net/softbell/bsh/dto/request/InterlockTokenDto.java
package net.softbell.bsh.dto.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : 연동 토큰 DTO
 */
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class InterlockTokenDto
{
	private Long memberInterlockId;
    private String name;
}

File: src/main/kotlin/net/softbell/bsh/dto/request/InterlockTokenDto.kt
package net.softbell.bsh.dto.request

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : 연동 토큰 DTO
 */
data class InterlockTokenDto (
        var memberInterlockId: Long? = null,
        var name: String
)